backend/graphql/schema.graphql

```diff
@@ -1,18 +1,50 @@
-"A date string with format `Y-m-d`, e.g. `2011-05-23`."
+"A date string wit`h format `Y-m-d`, e.g. `2011-05-23`."
 scalar Date @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\Date")
 
 "A datetime string with format `Y-m-d H:i:s`, e.g. `2018-05-23 13:43:32`."
 scalar DateTime @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")
 
 type Query {
+    "Return user accounts in the application."
     users: [User!]! @paginate(defaultCount: 10)
+
+    "Return details about a specific user in the application"
     user(id: ID @eq): User @find
 }
 
+"""
+Input type for userCreate mutation
+"""
+input CreateUserInput {
+    "Display name. (optional)"
+    name: String
+    
+    "Username. Must be unique. (required)"
+    username: String! @rules(apply: ["unique:users,username"],  messages: { unique: "Username is already in use"})
+    
+    "Email address. Must be unique. (required)"
+    email: String! @rules(apply: ["unique:users,email"], messages: { unique: "Email address is already in use"})
+    
+    "Password used to login to the application. (required)"
+    password: String! @hash
```

Comment:
We need to add checking password complexity with https://github.com/olssonm/laravel-zxcvbn

Check out the validation criteria in #60

Reply:
Added